ISSUE_TEMPLATE.md

```diff
@@ -1 +1,47 @@
+<!--
+Please use the below format to report the issue in a better way.
+To check any option, replace the space between the "[" and "]" with a "x".
+-->
 
+#### Nature of Issue ?
+
+- [ ] Found a Bug
+- [ ] Existing Feature Enhancement
+- [ ] New Feature Request
+
+#### Most Appropriate Sub Area ?
+
+- [ ] Color
+- [ ] Core
+- [ ] Events
+- [ ] Image
+- [ ] IO
+- [ ] Math
+- [ ] Typography
+- [ ] Utilities
+- [ ] WebGL
```

Comment:
I totally agree with your suggestion, and I did actually include a "Platform/Device" point under the "Details of Bug" part, but it wasn't so clear then. Added your suggestion as a little separate question below this to better help in understanding !